Tools/Scripts/webkitpy/port/watch_simulator.py

```diff
@@ -98,6 +98,3 @@ def developer_dir(self):
 
     def logging_patterns_to_strip(self):
         return []
-
-    def stderr_patterns_to_strip(self):
-        return []
```

Comment:
It feels inconsistent to have stdout here, but not stderr. Should subclasses call through to super class explicitly, so that platform specific patterns could be easily added?

Reply:
I think so, I'll remove these empty returns everywhere